stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/application/ChatInitializer.kt

```diff
@@ -5,33 +5,40 @@ import com.google.firebase.FirebaseApp
 import io.getstream.chat.android.client.ChatClient
 import io.getstream.chat.android.client.logger.ChatLogLevel
 import io.getstream.chat.android.client.notifications.handler.NotificationConfig
+import io.getstream.chat.android.client.notifications.handler.NotificationHandlerFactory
 import io.getstream.chat.android.core.internal.InternalStreamChatApi
 import io.getstream.chat.android.livedata.ChatDomain
 import io.getstream.chat.android.pushprovider.firebase.FirebasePushDeviceGenerator
-import io.getstream.chat.android.pushprovider.huawei.HuaweiPushDeviceGenerator
 import io.getstream.chat.ui.sample.BuildConfig
 import io.getstream.chat.ui.sample.R
+import io.getstream.chat.ui.sample.feature.HostActivity
+
+
 
 @OptIn(InternalStreamChatApi::class)
 class ChatInitializer(private val context: Context) {
 
     @Suppress("UNUSED_VARIABLE")
     fun init(apiKey: String) {
         FirebaseApp.initializeApp(context)
+        NotificationHandlerFactory.newMessageIntent = {
+                messageId: String,
+                channelType: String,
+                channelId: String,
+            ->
+            HostActivity.createLaunchIntent(context, messageId, channelType, channelId)
+        }
         val notificationConfig =
             NotificationConfig(
-                smallIcon = R.drawable.ic_chat_bubble,
-                loadNotificationDataIcon = R.drawable.ic_chat_bubble,
                 pushDeviceGenerators = listOf(
-                    HuaweiPushDeviceGenerator(context, ApplicationConfigurator.HUAWEI_APP_ID),
+                    // HuaweiPushDeviceGenerator(context, ApplicationConfigurator.HUAWEI_APP_ID),
```

Comment:
Are we disabling this on purpose?

Reply:
Ouch, it was because Firebase seems to be faster than Huawei and I didsabled it for testing purpose, let me enable it again